src/applications/vaos/services/appointment/index.js

```diff
@@ -125,9 +125,14 @@ export async function getBookedAppointments({
     if (useV2) {
       const appointments = await getAppointments(startDate, endDate, [
         'booked',
+        'cancelled',
       ]);
 
-      return transformVAOSAppointments(appointments);
+      const appointmentsWithoutRequests = appointments.filter(
+        appt => !appt.requestedPeriods,
```

Comment:
Items with a cancelled status could either be cancelled requests or cancelled appointments, so we need to filter out the ones we don't want depending on the function we're in.